.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: python:3.8
+    steps:
+    - checkout
+    - &build
+      run:
+        name: Build
+        command: |
+          python3.8 -m venv venv/
+          venv/bin/pip install --upgrade -r requirements.txt -r requirements_tests.txt
+    - run:
+        name: PyTest
+        command: venv/bin/pytest
```

Comment:
It would be nice to run the linter (flake8) as well. Installing tox into the global environment and then running tox could be a way to make sure the test configuration is only written once.

Reply:
I added flake8 and black (for enforcing code formatting). It'd probably also be good to add pydocstyle to enforce documentation, but that could be a follow-up PR.